ebpf.mk

```diff
@@ -9,6 +9,7 @@ build:
 		-Wno-unused-value \
 		-Wno-pointer-sign \
 		-Wno-compare-distinct-pointer-types \
+		-Wno-address-of-packed-member \
```

Comment:
We've recently been using clang pragmas to ignore these errors, see https://github.com/weaveworks/tcptracer-bpf/pull/51 for an example

Reply:
Done